src/treasury.cairo

```diff
@@ -29,6 +29,16 @@ trait ITreasury<TContractState> {
     fn get_amm_address(self: @TContractState) -> ContractAddress;
     fn deposit_to_zklend(ref self: TContractState, token: ContractAddress, amount: u256);
     fn withdraw_from_zklend(ref self: TContractState, token: ContractAddress, amount: u256);
+    fn deposit_eth_to_nostra_lending_pool(ref self: TContractState, amount: u256);
+    fn deposit_usdt_to_nostra_lending_pool(ref self: TContractState, amount: u256);
+    fn deposit_strk_to_nostra_lending_pool(ref self: TContractState, amount: u256);
+    fn deposit_usdc_to_nostra_lending_pool(ref self: TContractState, amount: u256);
+    fn deposit_wbtc_to_nostra_lending_pool(ref self: TContractState, amount: u256);
+    fn withdraw_eth_from_nostra_lending_pool(ref self: TContractState, amount: u256);
+    fn withdraw_usdt_from_nostra_lending_pool(ref self: TContractState, amount: u256);
+    fn withdraw_strk_from_nostra_lending_pool(ref self: TContractState, amount: u256);
+    fn withdraw_usdc_from_nostra_lending_pool(ref self: TContractState, amount: u256);
```

Comment:
Please please don't do this, this is heavy code duplication. Probably have the caller supply the token address to deposit/withdraw.